helm/kiam-app/values.yaml

```diff
@@ -33,6 +33,10 @@ agent:
 
   log:
     level: info
+    # Default gprc log level and verbosity, see https://github.com/grpc/grpc-go/blob/master/grpclog/loggerv2.go#L130
+    # Only change it to debug failing grpc calls, see https://github.com/uswitch/kiam/issues/115#issue-340009605
+    grpcLogLevel: error
```

Comment:
those are the default settings, which do not show more logs, however those become relevant in case of adjusting the grpcLogLevel to info and setting the grpcLogVerbosity to 8. The comments above are mainly to show how to set them in case you want to debug.